Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -107,6 +111,19 @@ class BrowserViewController: UIViewController {
         }
     }
 
+    override func viewWillTransitionToSize(size: CGSize, withTransitionCoordinator coordinator: UIViewControllerTransitionCoordinator) {
+        if let displayedPopoverController = displayedPopoverController {
```

Comment:
Nit: We can simplify this down to `displayedPopoverController?.dismissViewControllerAnimated(true, completion: nil)`